gnoi/gnoi.go

```diff
@@ -745,24 +752,642 @@ func (s *system) getSupervisorRole(ctx context.Context) (activeSupervisor, stand
 	}
 }
 
+type linkQualification struct {
+	plqpb.UnimplementedLinkQualificationServer
+
+	c      *ygnmi.Client
+	config *configpb.Config
+
+	mu sync.RWMutex
```

Comment:
It's idiomatic to note above this `mu` what it is protecting -- I assume that this is protecting both the `qualifications` and the `historicalResults` maps?